sql/database.sql

```diff
@@ -11001,3 +11001,25 @@ CREATE TABLE `api_token` (
     `expiry`       datetime NULL,
     PRIMARY KEY (`id`)
 ) ENGINE = InnoDB;
+
+DROP TABLE IF EXISTS `benefit_eligibility`;
```

Comment:
Should delete the benefit_eligibility table above rather than drop it.

Reply:
I assume the drop table here is fine but just that I forgot to delete eligibility_response table above.